webapp/packages/core-view/src/View/parseHotkey.ts

```diff
@@ -38,21 +54,21 @@ export function mapKey(key: string): string {
     .replace(/key|digit|numpad|arrow/, '');
 }
 
-export function isHotkeyModifier(key: string) {
+export function isHotkeyModifier(key: string): boolean {
   return reservedModifierKeywords.includes(key);
 }
 
 export function parseKeysHookInput(keys: string, splitKey = ','): string[] {
   return keys.split(splitKey);
 }
 
-export function parseHotkey(hotkey: string, combinationKey = '+', description?: string): Hotkey {
+export function parseHotkey({ hotkey, combinationKey = '+', description, useKey = false, isSequence = false }: IParseHotkeysArgs): IHotkeyParsed {
```

Comment:
It is accepting arguments that no longer exist, there are only two uses of this function, and each: `parseHotkey({ hotkey: key })`